apps/web/src/components/Player/PlayerMenuBar/PlayerMenuBar.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback, useRef, useState } from 'react';
 import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
 import { Typography, Grid, Button } from '@material-ui/core';
+import { isMobile } from '../../../utils';
```

Comment:
Will `<Hidden>` work instead?

Also we should not use this `isMobile` to adjust styles. Using the userAgent to adjust styles on mobile is not responsive - the UI cannot adjust itself in response to changes in browser width.